server/publications/collections/products.js

```diff
@@ -95,6 +95,15 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
     return this.ready();
   }
 
+  // Get disabled shop id's to use for filtering
+  const disabledShopIds = Shops.find({
```

Comment:
Shouldn't we do this the same way we did above? e.g. `activeShops` rather than `disabledShops`?